src/main/java/com/oltpbenchmark/benchmarks/tpcc/procedures/Delivery.java

```diff
@@ -104,7 +104,7 @@ public void run(Connection conn, Random gen, int w_id, int numWarehouses, int te
 
         int[] orderIDs = new int[10];
 
-        for (d_id = 1; d_id <= terminalDistrictUpperID; d_id++) {
+        for (d_id = 1; d_id <= w.getDistPerWhse(); d_id++) {
```

Comment:
Instead of getting the districts per warehouse from `TPCCWorker`, why not just use `TPCCConfig.configDistPerWhse` since it never changes?

Reply:
You're right, thank you! I've updated the pull request. And I'm sorry, that it took so long time.